packages/http-client/src/http-client.ts

```diff
@@ -121,6 +122,32 @@ export function execute<ReturnT>(executeFn: ExecuteHttpRequestFn<ReturnT>) {
   };
 }
 
+function encodeRequestQueryParameters(
+  parameter: Record<string, string> | undefined
+): Record<string, string> | undefined {
+  if (parameter) {
+    return Object.keys(parameter).reduce(
+      (encodedParams, key) => ({
+        ...encodedParams,
+        [key]: encodeURIComponent(parameter[key])
```

Comment:
```suggestion
        [encodeURIComponent(key)]: encodeURIComponent(parameter[key])
```
Could it be possible to have a special char in the key? Otherwise this should be fine.